teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+# define teacher
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
+
+  def can_use_services
+    true
+  end
```

Comment:
- [ ] The names of predicate methods (methods that return a boolean value) should end in a question mark. Please rename `can_use_services` to `can_use_services?`

Reply:
Alright, thanks @codepantha